core/templates/dev/head/components/solution_editor_directive.html

```diff
@@ -0,0 +1,63 @@
+<div style="height: 100%;">
+  <div ng-if="!solutionEditorIsOpen"
+       style="height: 100%;"
+       ng-attr-title="<[isEditable ? 'Edit solution' : '']>">
+    <div class="oppia-readonly-rule-tile"
+         ng-class="{'oppia-editable-section': isEditable}">
+      <div class="oppia-rule-preview oppia-transition-200">
+        <div class="oppia-click-to-start-editing"
+             ng-if="isEditable" ng-click="openSolutionEditor()">
+          <i class="material-icons oppia-editor-edit-icon pull-right"
+             title="Edit Solution">&#xE254;
+          </i>
+        </div>
+        <strong>
+          <[stateSolutionService.savedMemento.answerIsExclusive ? 'The only' : 'One']> solution is...
+        </strong>
+        <span angular-html-bind="stateSolutionService.savedMemento.getCorrectAnswerHtml(getObjectType())"></span>
```

Comment:
Why doesn't this reuse the correct answer display in the learner view (which is not based on object type)?